storage/watchable_store_test.go

```diff
@@ -222,3 +240,130 @@ func TestUnsafeAddWatching(t *testing.T) {
 		}
 	}
 }
+
+func TestNewMapWatchingToEventMap(t *testing.T) {
+	var (
+		testN         = 5
+		testKey       = []byte("Foo")
+		testValue     = []byte("Bar")
+		testKeys      = make([][]byte, testN)
+		testValues    = make([][]byte, testN)
+		testWatchings = make([]*watching, testN)
+		testEvents    = make([]storagepb.Event, testN)
+	)
+	for i := 0; i < testN; i++ {
```

Comment:
do we really need to populate keys and events in a for loop. I feel it is cleaner to just assign it by hand.